packages/web-components/tests/input.spec.ts

```diff
@@ -299,24 +299,25 @@ test.describe('Input with form', () => {
         //Assert
         await expect(spellCheckInput).toHaveAttribute('spellcheck', 'true')
     })
-    test('applies autocomplete prop to shadow input', async ({ page }) => {
-        //Arrange
-        const autocomplete = page.locator('#autocomplete')
-        const autocompleteInput = autocomplete.locator('input').nth(1)
-
-        //Assert
-        await expect(autocompleteInput).toHaveAttribute('autocomplete', 'on')
```

Comment:
Yeah, these were done with the assumption that if the attribute existed on the shadow dom'd input, then it would be a working attribute. You can see how that turned out 🙃